packages/provider/src/language-model/v2/language-model-v2-call-options.ts

```diff
@@ -131,4 +131,16 @@ Only applicable for HTTP-based providers.
    * functionality that can be fully encapsulated in the provider.
    */
   providerOptions?: SharedV2ProviderOptions;
+
+  /**
+   * Whether to use structured outputs.
+   *
+   * @default false
+   */
+  structuredOutputs?:
+    | {
+        tools?: boolean;
+        response?: boolean;
+      }
+    | boolean;
 };
```

Comment:
please revert this is not what i meant

Reply:
it should be a provider option for those models - not a generic option on generateText